ssBind/posepicker/PosePicker.py

```diff
@@ -190,48 +174,28 @@ def pick_poses(
+    
         for i in range(len(pcs)):
             for j in range(i + 1, len(pcs)):
-
-                r(
-                    f"""
-                library(ggplot2)
-                library(ggdensity)
-
-                df <- read.table('PCA_Scores.csv', header=TRUE, sep=",")
-                
-                p <- ggplot(df, aes(x={pcs[i]}, y={pcs[j]}, z=Score)) +
-                    stat_summary_2d(fun=mean, binwidth = {self._binsize}) +
-                    scale_fill_gradientn(colours = rainbow(5), limits = c(min(df$Score),max(df$Score))) +
-                    geom_hdr_lines() +
-                    labs(fill = "Score") +
-                    theme(
-                panel.border = element_rect(color = "black", fill = 'NA', linewidth = 1.5),
-                panel.background = element_rect(fill = NA),
-                panel.grid.major = element_line(linewidth = 0),
-                panel.grid.minor = element_line(linewidth = 0),
-                axis.text = element_text(size = 16, color = "black", face = "bold"),
-                axis.ticks = element_line(color = "black", linewidth = 1),
-                axis.ticks.length = unit(5, "pt"),
-                axis.title.x = element_text(vjust = 1, size = 20, face = "bold"),
-                axis.title.y = element_text(angle = 90, vjust = 1, size = 20, face = "bold"),
-                legend.text = element_text(size = 10, face="bold"),
-                legend.title = element_text(size = 16, hjust = 0, face="bold")
-                ) +
-                annotate("text", x = c("""
-                    + str(", ".join(map(str, pcx[i])))
-                    + """), y = c("""
-                    + str(", ".join(map(str, pcx[j])))
-                    + """), size = 6, 
-                fontface = "bold", label = c("""
-                    + str(", ".join(map(str, list(range(1, len(clusts) + 1)))))
-                    + f""")) +
-                coord_fixed((max(df${pcs[i]})-min(df${pcs[i]}))/(max(df${pcs[j]})-min(df${pcs[j]})))
-
-                ggsave("{pcs[i]}-{pcs[j]}.svg", width = 7, height = 7)
-
-                """
-                )
+                colors = df["Score"].to_numpy()
+                fig, ax = plt.subplots()
+                scatter = ax.scatter(df[f"{pcs[i]}"], df[f"{pcs[j]}"], s=0.5, c=colors, cmap="gist_rainbow")
+                sns.kdeplot(x=df[f"{pcs[i]}"], y=df[f"{pcs[j]}"], levels=[0.010, 0.050, 0.20, 0.50], bw_adjust=0.78,
+                            cmap=cmap1)
+                ax.set_xlabel(f"{pcs[i]}")
+                ax.set_ylabel(f"{pcs[j]}")
+                ax.set(xlim=(df[f"{pcs[i]}"].min(), df[f"{pcs[i]}"].max()), ylim=(df[f"{pcs[j]}"].min(), df[f"{pcs[j]}"].max()))
+                colorbar = fig.colorbar(scatter, cmap="gist_rainbow", label="Score", shrink=0.5)
+                for index, dict_i in enumerate(labels):
+                    coord_index = list(dict_i.values())[0]
+                    x_coord = df[f"{pcs[i]}"].iloc[coord_index]
```

Comment:
`iloc` is a bit risky in case the dataframe gets reindexed or reordered (should not be the case at the moment but it's easy to break). Since df has a column called `Index`, you should be able to do `df.loc[df.Index == coord_index, f"{pcs[i]}"]`